test/e2e/cypress/support/commands.js

```diff
@@ -55,9 +55,11 @@ Cypress.Commands.add('loadScenario', (scenario) => {
     Cypress.env('web_api_port'),
   ];
   cy.log(`Loading scenario "${scenario}"...`);
-  cy.exec(
-    `cd ${projectRoot} && ${photofinishBinary} run --url "http://${webAPIHost}:${webAPIPort}/api/collect" ${scenario}`
-  );
+  if(photofinishBinary) {
```

Comment:
Could you move the `cy.log` statement inside this `if` and add an `else` clause that tells the user that `photofinish` isn't being used?